app/src/main/java/com/example/footballmanager/ui/FootballManagerApp.kt

```diff
@@ -6,17 +6,20 @@ import androidx.compose.foundation.layout.padding
 import androidx.compose.material3.ExperimentalMaterial3Api
 import androidx.compose.material3.Scaffold
 import androidx.compose.material3.TopAppBar
+import androidx.compose.material3.TopAppBarColors
 import androidx.compose.material3.TopAppBarDefaults
 import androidx.compose.material3.TopAppBarScrollBehavior
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.LaunchedEffect
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.input.nestedscroll.nestedScroll
 import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.tooling.preview.Preview
 import androidx.lifecycle.viewmodel.compose.viewModel
 import com.example.footballmanager.screens.HomeScreen
-import com.example.footballmanager.screens.result_screens.success_components.Header
+import com.example.footballmanager.ui.theme.Header
 import com.example.footballmanager.screens.view_models.HomeViewModel
+import com.example.footballmanager.ui.theme.background
 
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
```

Comment:
it's better to user `hiltViewModel()` 
article: https://medium.com/@NickFan34818768/deep-dive-into-jetpack-compose-viewmodel-injection-89043f42b76d